ai/worker/docker.go

```diff
@@ -362,6 +363,7 @@ func (m *DockerManager) createContainer(ctx context.Context, pipeline string, mo
 	envVars := []string{
 		"PIPELINE=" + pipeline,
 		"MODEL_ID=" + modelID,
+		"HF_TOKEN=" + os.Getenv("HF_TOKEN"),
 	}
```

Comment:
```suggestion
}
if token := os.Getenv("HF_TOKEN"); token != "" {
	envVars = append(envVars, "HF_TOKEN="+token)
}
```

Reply:
I used the `os.LookupEnv` function for the same functionality: <https://pkg.go.dev/os#LookupEnv>